src/main/scala/io/iohk/ethereum/ledger/Ledger.scala

```diff
@@ -89,14 +83,18 @@ class LedgerImpl(vm: VM, blockchainConfig: BlockchainConfig) extends Ledger with
     */
   @tailrec
   private def executeTransactions(signedTransactions: Seq[SignedTransaction], world: InMemoryWorldStateProxy,
-                                  blockHeader: BlockHeader, config: EvmConfig, signedTransactionValidator: SignedTransactionValidator,
+                                  blockHeader: BlockHeader, signedTransactionValidator: SignedTransactionValidator,
                                   acumGas: BigInt = 0, acumReceipts: Seq[Receipt] = Nil): Either[TxsExecutionError, BlockResult] =
     signedTransactions match {
       case Nil =>
         Right(BlockResult(worldState = world, gasUsed = acumGas, receipts = acumReceipts))
 
       case Seq(stx, otherStxs@_*) =>
-        validateTransaction(stx, world, acumGas, blockHeader, signedTransactionValidator, config) match {
+        val senderAccount = world.getAccount(stx.senderAddress)
+        val validatedStx = senderAccount
+          .toRight(Left(TxsExecutionError(s"Account of tx sender ${Hex.toHexString(stx.senderAddress.toArray)} not found")))
```

Comment:
maybe we can use `io.iohk.ethereum.domain.Address#toString` to print the address